cmd/juju/application/deployer/bundlehandler.go

```diff
@@ -862,12 +862,6 @@ func (h *bundleHandler) addApplication(change *bundlechanges.AddApplicationChang
 
 	resMap := h.makeResourceMap(charmInfo.Meta.Resources, p.Resources, p.LocalResources)
 
-	if err := lxdprofile.ValidateLXDProfile(lxdCharmInfoProfiler{
```

Comment:
This isn't part of the DeploySteps. I'm debating whether removing it here is appropriate, this will be validated later. This PR does keep the one for local charms.